test/test_playbooks/job_template.yml

```diff
@@ -47,6 +47,50 @@
       job_template_state: absent
       expected_change: false
 
+  - include: tasks/job_template_from_file.yml
+    vars:
+      job_template_state: present
+      job_template_file_name: "delete_file.erb"
+      expected_change: true
+
+  - name: Add the same template input via direct input
+    include: tasks/job_template.yml
+    vars:
+      job_template_name: delete file
+      job_template_job_category: Commands
+      job_template_description_format: "delete %{file}"
+      job_template_locations: "{{ omit }}"
+      job_template_organizations: "{{ omit }}"
```

Comment:
I would prefer to always set the taxonomies by ansible parameters (this would closely resemble the common use case).